src/containers/Template/Form/SpeedSend/SpeedSend.test.tsx

```diff
@@ -52,15 +52,21 @@ describe('SpeedSend', () => {
       target: { value: 'new Template' },
     });
 
+    //TODO: add case to fill the draftjs input
+
     fireEvent.change(container.querySelector('input[name="languageId"]'), {
       target: { value: 1 },
     });
+
     const { queryByText } = within(container.querySelector('form'));
     const button = queryByText('Save');
     fireEvent.click(button);
     await wait();
     await wait();
-    const { getByText } = within(container.querySelector('tbody'));
-    expect(getByText('Good message')).toBeInTheDocument();
+
+    expect(queryByText('Message is required.')).toBeInTheDocument();
+
+    // const { getByText } = within(container.querySelector('tbody'));
```

Comment:
should we remove this if not needed?

Reply:
We should revisit this once we add TODO on line 55.